src/Core/Actions/ResendMailAction.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace Basement\BetterMails\Core\Actions;
+
+use Basement\BetterMails\Core\DTOs\ResendMailDTO;
+use Illuminate\Mail\Message;
+use Illuminate\Support\Facades\Mail;
+
+final class ResendMailAction
+{
+    public function send(ResendMailDTO $dto): void
+    {
+        Mail::send([], [], function (Message $message) use ($dto) {
```

Comment:
// TODO: enhance architecture to store Notifiable/Mailable to be able to resend in the same terms.